index.html

```diff
@@ -653,9 +720,8 @@ <h2>
           window).
         </dd>
         <dd>
-          The <code>browser</code> <a>display mode</a> doesn't have a
-          <a>fallback display mode</a> (conforming user agents are required to
-          support the <code>browser</code> <a>display mode</a>).
+          Conforming user agents are required to
```

Comment:
I think you can just delete this paragraph. It's redundant for the sentence above: "As such, the user agent MUST support the browser display mode."

I think it was redundantly here before just to explain why it's OK to not have a fallback display mode. That's equivalent to the new note that I suggested you add above.